### ISSUE TITLE
Blank screen on Admin Panel

### ISSUE THREAD

**Issue by vamshi9** [keystonejs/keystone#4585]
If I make any further changes in models, I am getting a blank screen in the admin panel but not user panel. But the status shows 200. I am using mlab connection for the database. What would be the possible reason for this?

**Model.js**
```
var keystone = require('keystone');
    Types = keystone.Field.Types;
   
var Teaching = new keystone.List('teaching',{
    autokey : {path : 'slug', from : 'title' , unique : 'true'},
    map : {name : 'title'},
    defaultSort : '-createdAt'
});    

Teaching.add({
    title : {type : String, required :true},
    course : {type : String},
    professorName : {type : String}
});

/*virtuals for semester here*/

Teaching.defaultcolumns = 'title , course , professorName';
Teaching.register();
```
**Keystone.js**
```

require('dotenv').config();

var keystone = require('keystone');
var handlebars = require('express-handlebars');
var middleware = require('./routes/middleware');

// if(middleware.userInfo.name.first == "HOD"){
// 	console.log("correct stategy");
// }

keystone.init({
	'name': 'DMS',
	'brand': 'DMS',

	'sass': 'public',
	'static': 'public',
	'favicon': 'public/favicon.ico',
	'views': 'templates/views',
	'view engine': '.hbs',
	'mongo': process.env.MONGO_URI,

	'custom engine': handlebars.create({
		layoutsDir: 'templates/views/layouts',
		partialsDir: 'templates/views/partials',
		defaultLayout: 'default',
		helpers: new require('./templates/views/helpers')(),
		extname: '.hbs',
	}).engine,

	'emails': 'templates/emails',

	'auto update': true,
	'session': true,
	'auth': true,
	'user model': 'User',
});

keystone.import('models');

keystone.set('admin path','admin');
keystone.set('locals', {
	_: require('lodash'),
	env: keystone.get('env'),
	utils: keystone.utils,
	editable: keystone.content.editable,
});

keystone.set('routes', require('./routes'));

keystone.set('cloudinary config', process.env.CLOUDINARY_URL);
keystone.set('cookie secret', process.env.COOKIE_SECRET);
    //keystone.set('mailgun api key', process.env.MANDRILL_API_KEY);

keystone.set('nav', {
	posts: ['posts', 'post-categories'],
	galleries: 'galleries',
	enquiries: 'enquiries',
	users: 'users',
	research : 'research',
	publications : 'publications',
	teachingInitiatives : 'teaching'
});

keystone.set('signin logo','../images/logo.png');
keystone.set('signin url','/');
keystone.set('signin redirect','/');

// if (!process.env.MAILGUN_API_KEY || !process.env.MAILGUN_DOMAIN) {
// 	console.log('----------------------------------------'
// 	+ '\nWARNING: MISSING MAILGUN CREDENTIALS'
// 	+ '\n----------------------------------------'
// 	+ '\nYou have opted into email sending but have not provided'
// 	+ '\nmailgun credentials. Attempts to send will fail.'
// 	+ '\n\nCreate a mailgun account and add the credentials to the .env file to'
// 	+ '\nset up your mailgun integration');
// }

keystone.set('port',7000);
keystone.start();

```

**Comment by radames** [comment]
hi @vamshi9 this is related to your custom admin path,  #4567 and there is a temp solution editing the sagas Admin react app 

https://github.com/radames/keystone/commit/f10a9233df4c14ce1df2f8c1839255f727bb2b2b#diff-f2d2df298511cc65f89c3f74048449f7L71

**Comment by Noviny** [comment]
Solved in #4567
